### ISSUE TITLE
Mark Slack logo as decorative with aria-hidden="true"

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/areweheadlessyet#48]
This will improve the accessibility of the Slack CTA in the header.

---

Assigning to @shyusu4 as part of her Outreachy project.
